From 0ab5747185be9135b0711e72fc64dfa6a33f3fd3 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@debian.org>
Date: Wed, 17 Dec 2025 01:07:49 +0000
Subject: [PATCH] Fix configuration for pytest 9 (#74)

Before this change, pytest 9 failed with:

```
TypeError: pyproject.toml: config option 'addopts' expects a list for type 'args', got str: '--cov --cov-report=html --cov-fail-under=100'
```

See https://github.com/pytest-dev/pytest/issues/13743.
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 87ee999..3df3111 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -39,4 +39,4 @@ build-backend = "hatchling.build"
 path = "expandvars.py"
 
 [tool.pytest]
-addopts = '--cov --cov-report=html --cov-fail-under=100'
+addopts = ["--cov", "--cov-report=html", "--cov-fail-under=100"]
